client/components/common/UserProfile.tsx

```diff
@@ -0,0 +1,160 @@
+/** @jsx jsx */
+
+import React from 'react';
+import { jsx } from '@emotion/core';
+import { useTranslation } from 'i18n';
+import { useTheme } from 'emotion-theming';
+import { mq } from 'common/constants';
+import { Button } from 'antd';
+import { EditOutlined } from '@ant-design/icons';
+import BuildList from './BuildList';
+import ProfilePictureModal from './ProfilePictureModal';
+import { Theme } from 'common/types';
+import { getImageUrl } from 'common/utils';
+
+interface Props {
+  username: string;
+  creationDate: string;
+  profilePicture: string;
+  isEditable: boolean;
+}
+
+const UserProfile: React.FC<Props> = ({
+  username,
+  creationDate,
+  profilePicture,
+  isEditable,
+}) => {
+  const [pictureModalVisible, setPictureModalVisible] = React.useState(false);
+
+  const { t } = useTranslation('common');
+  const theme = useTheme<Theme>();
+
+  const getDate = (dateStr: string) =>
+    new Date(dateStr).toLocaleDateString(undefined, {
+      month: 'short',
+      day: 'numeric',
+      year: 'numeric',
+    });
+
+  return (
+    <div
+      css={{
+        display: 'flex',
+        flexDirection: 'column',
+        maxWidth: 1036,
+        width: '100%',
+        margin: '0 auto',
+        [mq[1]]: { flexDirection: 'row' },
+      }}
+    >
+      <div
+        css={{
+          display: 'flex',
+          flexDirection: 'row',
+          margin: '0px 20px 20px 0px',
+          gridArea: '1 / 3 / 1 /1',
```

Comment:
This is not used, right?

Reply:
After having made the change to centered image, no. Removed.